history/characters/200000_farraki.txt

```diff
@@ -8632,3 +8634,57 @@
 	635.8.7={ birth=yes }
 	718.9.14={ death=yes }
 }
+ruuzlu = { # right-hand man and friend of Ukorz Sandscalp
+	name = Ruuzlu
+	culture=farraki religion=farrism
+	martial=8 diplomacy=6 stewardship=5 intrigue=9 learning=4
+	trait=education_intrigue_3 trait=diligent trait=sadistic trait=deceitful
+	586.1.30={ birth=yes }
+	592.08.13={
+		employer = 200104 # Ukorz
+	}
+	596.1.30={
+		set_relation_friend = character:200104 # Ukorz Sandscalp
+	}
+	653.9.30={ death=yes }
+}
+sul = { # member of the Council of Elders
+	name = Sul
+	culture=farraki religion=farrism
+	martial=7 diplomacy=5 stewardship=5 intrigue=8 learning=7
+	trait=education_intrigue_3 trait=arrogant trait=ambitious trait=diligent
+	538.8.29={ birth=yes }
+	563.8.29={
+		give_nickname = nick_the_sandcrawler
+	}
+	613.7.5={ # slain by adventurers in Throne of Thunder
+		death = {
+			death_reason = battle
+		}
+	}
+}
+jakrazet = {
+	name = Jakra'zet
+	culture = farraki
+	religion = farrism
+	martial=9 diplomacy=5 stewardship=5 intrigue=8 learning=5
+	trait=education_martial_4 # his strategic thinking earned him the position of General
+	trait=desert_warrior
+	trait=diligent trait=deceitful trait=zealous
+	disallow_random_traits = yes
+	553.8.19 = { birth= yes }
+	613.1.13 = { # joined Prophet Zul during the Pandaria campaign
+		employer = 66008 # Zul
+	}
+	613.12.10 = { # promoted to General of the Zandalari army after the Pandaria campaign
+		employer = 66000 # Rastakhan
```

Comment:
So this will make him marshal. Do you want other characters to be councilors as well?
```suggestion
		employer = 66000 # Rastakhan
		give_council_position = councillor_marshal
```

Reply:
> Do you want other characters to be councilors as well?

I think it would make sense if Vilnak'dor was the Marshal before Jakra'zet is promoted. I'll add it.